src/components/core/update/updateSlides.js

```diff
@@ -15,7 +15,7 @@ export default function updateSlides() {
   const slidesGrid = [];
   const slidesSizesGrid = [];
 
-  function slidesForMargin(slideEl, slideIndex) {
+  function slidesForMargin(slideIndex) {
```

Comment:
It can not be deleted, now `slideIndex` will represent the slide element and it will work wrong

Reply:
Sorry, I will restore it.